### PR TITLE
Document off switch

### PR DESCRIPTION
**Pull request by dragon-dxw**

### CODE REVIEW

**Comment by dragon-dxw** [comment]
[Document how to turn enrichment on / off](https://trello.com/c/vrPomN2N/475-document-how-to-turn-enrichment-on-off)

File: README.md [hunk 1]
```diff
@@ -150,3 +150,26 @@ Here are some brief notes on extending the infrastructure.
 - Adding an S3 bucket is done by invoking the `secure_bucket` module, located at `modules/secure_bucket/`, you can see how the existing buckets are created by viewing `modules/lambda_s3/bucket.tf`, new buckets should be created by adding to this file.
   If a bucket policy is added, then an extra statement will automatically be added that denies insecure transport.
 - Docker images are stored in ECR. Each repo needs to exist before a docker image can be pushed to ECR. These are created in `modules/lambda_s3/lambda.tf`.
+
+## 9 Turning Enrichment Off
+
```
Comment (anthonyhashemi) [inline README.md:155]: Do you think it's worth having the option of programatically turning off the lambdas in the source code here?

Also might be worth noting that any changes to lambdas will be overwritten by terraform when a new merge is made to main (of course not including secrets).

Comment (dragon-dxw) [reply README.md:155]: Added

**Review by anthonyhashemi — APPROVED** [review]
Sorry thought I had already approved!

**Pull request merged**